packages/node_modules/@ciscospark/redux-module-media/src/actions.js

```diff
@@ -201,12 +202,10 @@ export function acceptIncomingCall(incomingCall, spark) {
   return (dispatch) => {
     spark.phone.createLocalMediaStream()
       .then((localMediaStream) => {
+        dispatch(storeCall(incomingCall));
```

Comment:
Gotcha, yea, we definitely need more error handling. I'm hoping that we get designs for that soon so we can build out the UI for it.